AUTHORS

```diff
@@ -4,17 +4,38 @@ partial, alphabetical list of developers and contributors to Cantera over the
 years. If you've been left off, please report the omission on the Github issue
 tracker.
 
+Emil Atz
 Philip Berndt
+Tilman Bremer
 Victor Brunini, Sandia National Laboratory
 Bang-Shiuh Chen, Purdue University
+Ryan Crisanti
+Nicholas Curtis
 Steven DeCaluwe, Colorado School of Mines
+Vishesh Devgan
 Thomas Fiala, Technische Universität München
 David Fronczek
+@g3bk47
+Matteo Giani
 Dave Goodwin, California Institute of Technology
 John Hewson, Sandia National Laboratory
+Trevor Hickey
+Yuanjie Jiang
+JonKrisotfer
```

Comment:
Is this a Github username, or a missing space?

Reply:
Fixed